tests/virtctl/create_instancetype.go

```diff
@@ -0,0 +1,128 @@
+package virtctl
+
+import (
+	"context"
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	v1 "kubevirt.io/api/core/v1"
+	instancetypev1alpha2 "kubevirt.io/api/instancetype/v1alpha2"
+	generatedscheme "kubevirt.io/client-go/generated/kubevirt/clientset/versioned/scheme"
+	"kubevirt.io/client-go/kubecli"
+
+	. "kubevirt.io/kubevirt/pkg/virtctl/create/instancetype"
+	"kubevirt.io/kubevirt/tests/clientcmd"
+	"kubevirt.io/kubevirt/tests/decorators"
+	"kubevirt.io/kubevirt/tests/util"
+)
+
+const namespaced = "--namespaced"
+
+var clusterInstancetype *instancetypev1alpha2.VirtualMachineClusterInstancetype
+
+var _ = Describe("[sig-compute] create instancetype", decorators.SigCompute, func() {
+	var virtClient kubecli.KubevirtClient
+
+	BeforeEach(func() {
+		var err error
+		virtClient, err = kubecli.GetKubevirtClient()
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	AfterEach(func() {
+		if clusterInstancetype != nil {
```

Comment:
Can this leak objs when a test fails?

Reply:
`AfterEach` should run even after the failing test case, so it should not.